app/assets/javascripts/admin/modules/ga4-button-setup.js

```diff
@@ -1,6 +1,8 @@
 'use strict'
 window.GOVUK = window.GOVUK || {}
-window.GOVUK.Modules = window.GOVUK.Modules || {}
+window.GOVUK.analyticsGa4 = window.GOVUK.analyticsGa4 || {}
+window.GOVUK.analyticsGa4.analyticsModules =
+  window.GOVUK.analyticsGa4.analyticsModules || {}
```

Comment:
I wonder if it would be better to add a guard to each of the `init` functions, like:

```
Ga4ButtonSetup.prototype.init = function () {
    return if !analyticsIsEnabled // note - not sure what the variable to use would be here

    ...
```

Reply:
That was my first thought, but actually I think the only way I could do that would be to read the domain config and check the location myself, which is a bit painful especially considering the GA4 code in publishing components already does that.

Plus why load the modules at all if GA4 isn't enabled? These are purely GA4 analytics modules so I think having the GA4 code load them probably makes sense

Reply:
Yep, no objections from me 👍🏻